### ISSUE TITLE
It's been 150 days since I bought some ether, balance is still 0

### ISSUE THREAD

**Issue by shlomogoldblatt** [ethereum/mist#3137]
This is my address: 0xf794fB00f06FBD673cE3DB5EB0339c1e78dD0c3b
It shows my balance of 0.05 ether
but when I open Ether wallet, it is 0 ether.
I've downloaded all the new, updated versions, I updated blocks but for 5 months I'm having the same problem.

Can anyone tell me what is going up with this wallet?

**Comment by evertonfraga** [comment]
Please do the following:
- update to Mist or Ethereum Wallet 0.9.1
- update your Geth to the latest (as seen on the image below)

![image](https://user-images.githubusercontent.com/47108/31338308-0d4c34f2-acd5-11e7-802e-1c1a9d148e53.png)

- enable the light client:

![image](https://user-images.githubusercontent.com/47108/31338324-1d1beefe-acd5-11e7-8c6d-0eb002a3822a.png)

**Comment by mkoistinen** [comment]
[Etherscan](https://etherscan.io/address/0xf794fB00f06FBD673cE3DB5EB0339c1e78dD0c3b) shows that the above-mentioned account was emptied nearly 3 days after this was posted, so, I'm thinking this can be closed.

**Comment by shlomogoldblatt** [comment]
yes. I finally did it using myetherwallet. but ethereum/mist still have this problem and a lot of people are complaining about it
